docstr_coverage/configFile.py

```diff
@@ -0,0 +1,43 @@
+"""
+This module is for conf file .docstr.yaml
+"""
+import os
+
+import yaml
+
+
+def readConfigFile(paths, kwargs):
+    """
+    Read config file and assign all variable to paths, kwargs.
+
+    :param tuple paths: paths list
+    :param dict kwargs: options
+
+    :return (paths, kwargs)
+    :rtype (tuple, dict)
+    """
```

Comment:
Can we convert this to a NumPy-formatted docstring, please?
```suggestion
    """Read config file and assign all variable to paths, kwargs

    Parameters
    ----------
    paths: Tuple
        Filepaths 
    **kwargs: Dict
        Options

    Returns
    -------
    paths: Tuple
    kwargs: Dict
    """
```